swarm/src/connection/pool.rs

```diff
@@ -758,27 +762,34 @@ where
 
                     let (command_sender, command_receiver) =
                         mpsc::channel(self.task_command_buffer_size);
+                    let (event_sender, event_receiver) = mpsc::channel(self.task_event_buffer_size);
```

Comment:
Commenting here to make sure we document the changed behavior of `task_event_buffer_size`, i.e. from a size across all connections to the size of a single connection.

Reply:
I added a changelog entry bumped the version. Given that no other crates depend on this behaviour, I opted to not bump and dependent crates.

Reply:
I think the docs on `SwarmBuilder::connection_event_buffer_size` still need to be updated:
https://github.com/libp2p/rust-libp2p/blob/1b6715206732b1695f353c64e0f269aebf0aab7b/swarm/src/lib.rs#L1566-L1589

Reply:
Done!